test-count.js

```diff
@@ -0,0 +1,327 @@
+const fs = require("fs");
+const readline = require("readline");
+const path = require("path");
+const scenario = "Scenario";
+const manual = "@manual";
+const regres = "@regression";
+const smoke = "@smoke";
+const todo = "@to-do";
+const broken = "@broken-test";
+const unverified = "@un-verified";
+const precondition = "@pre-condition";
+const tags = [
+  "pre-condition",
+  "un-verified",
+  "broken-test",
+  "e2e",
+  "to-do",
+  "manual",
+  "regression",
+  "smoke",
+];
+const FILENAME =
+  typeof __filename !== "undefined"
+    ? __filename
+    : (/^ +at (?:file:\/*(?=\/)|)(.*?):\d+:\d+$/m.exec(Error().stack) || "")[1];
```

Comment:
I've searched why `__filename` might be undefined and found that this variable is available since Node.js 12+. Or code depends on Node.js 16 at the moment, so there is no reason for the fallback here.